README.md

```diff
@@ -99,6 +98,5 @@ function handleFind(json|error returned) {
     } else {
         io:println("find failed: " + returned.reason());
```

Comment:
Better to change `io:println` to `log:printInfo`

Reply:
Fixed